### ISSUE TITLE
Doesn't support multiple values for keys

### ISSUE THREAD

**Issue by mythmon** [JankGaming/vdfjs#1]
The VDF files I'm hoping to parse (Steam Controller configs) have multiple values for a single key. This data is mostly erased when parsing.

Here is a (greatly trimmed) example:

``` vdf
"controller_mappings"
{
    "group"
    {
        "id"        "0"
        "mode"      "four_buttons"
    }
    "group"
    {
        "id"        "1"
        "mode"      "dpad"
    }
    "group"
    {
        "id"        "2"
        "mode"      "dpad"
    }
    "preset"
    {
        "id"        "0"
        "name"      ""
    }
}
```

**Comment by twisterghost** [comment]
Thanks for the info, I will look into it asap.

**Comment by twisterghost** [comment]
Fixed in version 1.1.0 by #2
